trinity/utils/logging.py

```diff
@@ -8,30 +8,44 @@
 
 from typing import Tuple, Callable
 
+from trinity.config import (
+    ChainConfig,
+)
 
-def setup_trinity_logging(level: int) -> Tuple[Logger, Queue, handlers.QueueListener]:
+
+def setup_trinity_logging(chain_config: ChainConfig, level: int) -> Tuple[Logger, Queue, handlers.QueueListener]:
     from .mp import ctx
 
     log_queue = ctx.Queue()
 
     logging.basicConfig(level=level)
     logger = logging.getLogger('trinity')
 
-    handler = logging.StreamHandler(sys.stdout)
+    handler_stream = logging.StreamHandler(sys.stdout)
+    handler_file = logging.handlers.RotatingFileHandler(
+        str(chain_config.logfile_path), maxBytes=(1000000 * 5), backupCount=7
```

Comment:
@pipermerriam what do you feel about this number?

Reply:
I'd bump it up to 50mb filesize and 10 backups (max space 500mb).  Non trivial amount of storage but debug logs are going to be pretty verbose.